### PR TITLE
Use dwave-samplers>=1.0.0

### PR DESCRIPTION
**Pull request by arcondello**

### CODE REVIEW

**Review by randomir — APPROVED** [review]

File: setup.cfg [hunk 1]
```diff
@@ -23,13 +23,14 @@ classifiers =
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
```
Comment (randomir) [inline setup.cfg:26]: Perhaps add 3.11 to CI?

Comment (arcondello) [reply setup.cfg:26]: Unfortunately the orb CI doesn't work with 3.11 currently

Comment (randomir) [reply setup.cfg:26]: Oh? Is there an example I can take a look?

Comment (arcondello) [reply setup.cfg:26]: https://app.circleci.com/pipelines/github/dwavesystems/dwave-neal/122/workflows/0f58c362-fff1-4ec3-b86d-c5dc4ee831a8

Comment (randomir) [reply setup.cfg:26]: That should be an easy fix. I'll do it later.

Comment (arcondello) [reply setup.cfg:26]: IMO we should add CI, but I don't think it needs to block these since we test dwave-samplers with 3.11 already.

Comment (randomir) [reply setup.cfg:26]: I agree.

**Pull request merged**